docs/source/visualization/interval_color_mapping.ipynb

```diff
@@ -0,0 +1,328 @@
+{
```

Comment:
<p>Pondering if <code>buckets</code> is the right term to use here, usually bins is what people call it. Also worried about confusion over <code>buckets</code> being where data is stored.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/155/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>